app/views/NewMessageView.tsx

```diff
@@ -295,10 +294,14 @@ class NewMessageView extends React.Component<INewMessageViewProps, INewMessageVi
 		if (search.length === 0 && index === chats.length - 1) {
 			style = { ...style, ...sharedStyles.separatorBottom };
 		}
+
+		const itemSearch = item as ISearch;
+		const itemModel = item as TSubscriptionModel;
+
 		return (
 			<UserItem
-				name={item.search ? item.name : item.fname}
-				username={item.search ? item.username : item.name}
+				name={itemSearch.search ? itemSearch.name : itemModel.fname || ''}
+				username={itemSearch.search ? itemSearch.username : itemModel.name}
```

Comment:
It feels awkward to me.
I tried playing with type guards, but I couldn't do something better without creating a new util function.

@GleidsonDaniel do you have something?

Reply:
This I did together with @GleidsonDaniel . Was the way that we found